projects/objectnav_baselines/experiments/ithor/objectnav_ithor_base.py

```diff
@@ -53,7 +53,7 @@ def __init__(self):
         )
 
         self.NUM_PROCESSES = 80
-        self.TRAIN_GPU_IDS = [0, 1, 2, 3, 4, 5, 6]
+        self.TRAIN_GPU_IDS = [0, 1, 2, 3, 4, 5, 6, 7]
```

Comment:
We shouldn't be spawning 80 processes as, like you noted, this sometimes breaks things. Also we should presumably use `torch.cuda.device_count()` here so that we don't implicitly assume people have more resources than they do. This applies to `VALID_GPU_IDS` and `TEST_GPU_IDS` below as well.

Reply:
@Lucaweihs Agreed